src/docs/guides/fastify.md

```diff
@@ -15,30 +15,32 @@ This guide covers how to deploy a Fastify app on Railway in four ways:
 
 [![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/ZZ50Bj)
 
-You can also choose from a <a href="https://railway.app/templates?q=fastify" target="_blank">variety of Fastify app templates</a> created by the community.
+We highly recommend that [you eject from the template after deployment](/guides/deploy#eject-from-template-repository) to create a copy of the repo on your GitHub account.
+
+**Note:** You can also choose from a <a href="https://railway.app/templates?q=fastify" target="_blank">variety of Fastify app templates</a> created by the community.
 
 ## Deploy from a GitHub Repo
 
 To deploy a Fastify app on Railway directly from GitHub, follow the steps below:
 
-1. Fork the basic <a href="https://github.com/railwayapp-templates/fastify-hello-world" target="_blank">fastify-hello-world GitHub repo</a>.
+1. Fork the basic <a href="https://github.com/railwayapp-templates/fastify" target="_blank">fastify GitHub repo</a>.
 2. Create a <a href="https://railway.app/new" target="_blank">New Project.</a>
 3. Click **Deploy from GitHub repo**.
-4. Select your `fastify-hello-world` repo.
+4. Select your `fastify` repo.
     - Railway requires a valid GitHub account to be linked. If your Railway account isn't associated with one, you will be prompted to link it.
 5. Click **Deploy Now**.
 
-Once your Fastify app is successfully deployed, a Railway service (also referred to as an app) will be created for you. By default, this service will not be publicly accessible.
+Once your Fastify app is successfully deployed, a Railway [service](/guides/services) (also referred to as an app) will be created for you. By default, this service will not be publicly accessible.
```

Comment:
I think it would be best to stick to the conventional naming of just "service"

Reply:
In your experience, have you ever had users refer to it as an app or get confused between a service and app?

Reply:
Yeah, people call a service all kinds of things, so we don't want to also be using incorrect terminology.

Reply:
Gotcha!